lessons/04-blast.md

```diff
@@ -0,0 +1,121 @@
+One way to do so is to use the transdecode program which is included as a part of the Trinity package: http://trinityrnaseq.github.io/
+The logic behind this step is to filter all the contigs that have "junk" tails, for example one contig can be constructed from 40% CDS and 60% "junk" DNA. In that way we could latter reach  
+better results at the blast stage (It is also recommended to re-aligned the reads against these CDSs and not the original contigs in order to get real expression based on the CDS and not 
+on the tails).
+
+The trinity Transdecode package gives as an output both CDSs sequences file and proteins sequences file. For our propose we will use the proteins file.
+We will run it at the Blastp program (as demonstrated in the blast lesson) defining the output format parameter to 6 (-outfmt 6). 
+To read more on the different blast output formats follow these links:
+http://www.ncbi.nlm.nih.gov/books/NBK279690/
+http://www.ncbi.nlm.nih.gov/books/NBK279675/
+
+The are 12 columns at the blastp output, however we will not use all of them for the filtering steps. Their titles are as follows:
+
+1. Query Seq-id: normally the accession number of your contig or predicted protein.
+2. Subject Seq-id: the accession number of the sequence that got as hit from the protein database (can be swissprot, uniprot, etc')	�
+3. percent identity: global alignment %ID between both sequences
+4. alignment length
+5. number of mismatches
+6. number of gap openings
+7. query start
+8. query end
+9. subject start
+10. subject end
+11. Expect value (e-value)
+12. HSP bit score
+
+An exmple for the first blast output lines is (each column is separated by a tab):
+
+cds.comp100010_c1_seq1|m.67288	sp|Q9TW32|PPIB_DICDI	49.35	77	36	2	49	125	26	99	6.00E-12	67
+cds.comp100052_c0_seq1|m.67310	sp|Q5TYW6|RSPH9_DANRE	33.46	269	164	5	25	283	14	277	4.00E-37	138
+cds.comp100055_c0_seq1|m.67313	sp|P35248|SFTPD_RAT	45.24	84	43	1	137	217	56	139	1.00E-07	56.2
+cds.comp100073_c0_seq1|m.67327	sp|Q9M1K2|PI5K4_ARATH	37.24	196	122	1	9	204	70	264	4.00E-31	125
+cds.comp100100_c0_seq1|m.67345	sp|Q2QL79|CAV1_DIDVI	38.4	125	76	1	7	130	48	172	2.00E-25	100
+cds.comp100117_c0_seq2|m.67368	sp|Q5XEP2|HSOP2_ARATH	40.78	103	61	0	359	461	1	103	3.00E-17	88.2
+
+The next stage is to filter out the blast output file. If you are working on sequencing data it is recommended to work just on the first result for each query sequence. If you are working 
+just on few sequences you have the privilage to look at all the results and decide which is better.
+
+The common mistake within scientists is that they filter the results just by e-value and percent of ID. For examples all the alignments that have %ID>70 and e-value<0.00005 will be considered
+as potential homolgs. From first looking it sounds great, but the major question that should have be asked is what about the query coverage???

```

Comment:
* homolgs -> homologs
* From first looking -> On first glance* 
* should have be -> should have been